### ISSUE TITLE
When multiple option is set, placeholder defaults to "Select"

### ISSUE THREAD

**Issue by totalscoccia** [manishjanky/ngx-select-dropdown#31]
The following part of the code
```

if (this.multiple && this.selectedItems.length > 0) {
      this.selectedDisplayText = this.selectedItems.length === 1 ? text :
        text + ` + ${this.selectedItems.length - 1} more`;
    } else {
      this.selectedDisplayText = this.selectedItems.length === 0 ? this.config.placeholder : text;
    }

```
has the problem that when the placeholder is changed in the component attribute and the multiple option is set, this will default back to the value defined at line 66 (  public selectedDisplayText: string = "Select"; )

**Comment by totalscoccia** [comment]
Proposed pull fix in #32

**Comment by manishjanky** [comment]
please update to `v0.7.0`
